drivers/pwm/Kconfig.stm32

```diff
@@ -3,135 +3,11 @@
 # Copyright (c) 2016 Linaro Limited.
 # SPDX-License-Identifier: Apache-2.0
 
-menuconfig PWM_STM32
+config PWM_STM32
 	bool "STM32 MCU PWM driver"
 	depends on SOC_FAMILY_STM32
-	select USE_STM32_HAL_TIM
+	select USE_STM32_LL_TIM
 	help
 	  This option enables the PWM driver for STM32 family of
 	  processors. Say y if you wish to use PWM port on STM32
 	  MCU.
-
-if PWM_STM32
-
-config PWM_STM32_1
-	bool "STM32 PWM 1 Output"
-	help
-	  Enable output for PWM1 in the driver. Say y here
-	  if you want to use PWM1 output.
-
```

Comment:
Removing these symbols require to update whole set of boards using them.
Please do the board change accordingly.

Reply:
Thanks, will do. I preferred to wait until an initial review before doing that as noted in the PR notes. Before doing that, I'd also like your feedback on the question above (regarding `DT_HAS_NODE(DT_NODELABEL(timers12))`).

Reply:
Oups I didn't want this to be put as a single comment.
So indeed, we need to sort the labeling issue first.